app/workers/zync_worker.rb

```diff
@@ -139,7 +139,7 @@ def on_complete(_bid, options)
     # The number of retries is usually controlled at the origin of the failure using ThreeScale::SidekiqRetrySupport::Worker#last_attempt?,
     # but in this case we are not really using Sidekiq retries, but rather re-enqueueing the job manually here
     manual_retry_count = options['manual_retry_count'].to_i + 1
-    perform_async(event.event_id, event.data, manual_retry_count)
+    perform_async(event.event_id, event.data.to_json, manual_retry_count)
```

Comment:
Fixed a parameter not being sent as string to Redis. I wonder why this haven't reported errors to Bugsnag in production, as it's deprecated since Redis 6.4